frontend/public/components/import-yaml.jsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { connectToFlags, FLAGS } from '../features';
+import { PodModel } from '../models';
+import { CreateYAML } from './create-yaml';
+
+class ImportYamlPage_ extends React.PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const match = {url: '', params: {ns: 'default', plural: 'pods'}, isExact: true, path: ''};
+    return <React.Fragment>
+      <div className="yaml-editor-header">
+        <div>Import YAML</div>
+        <div className="yaml-editor-subheader">Create or replace resources from their YAML or JSON definitions. Use the template below or browse to upload a file.</div>
```

Comment:
Also not really sure what the `... browse to upload a file` should mean, since there is no `Browse` button.